charts/zwave2mqtt/values.yaml

```diff
@@ -7,7 +7,7 @@ strategyType: Recreate
 
 image:
   repository: robertslando/zwave2mqtt
-  tag: 3.0.1
+  tag: 4.0.3
```

Comment:
This version caused some issues with excessive logging. I suggest not updating this chart just yet.

Reply:
This seems reasonable to me... 
![image](https://user-images.githubusercontent.com/213795/91667245-20592900-ead1-11ea-8f23-478201bb18f2.png)

Reply:
Any chance your log file wasn't pruned after upgrading?

Reply:
It does get pruned.  This is what I see after upgrading to 4.0.3.  It seems to get into an endless logging loop after changing the power state of a zwave switch.  I'll  log an issue over there.  Maybe it's a problem that only affects certain setups:

![image](https://user-images.githubusercontent.com/6393612/91728314-eb9cae80-eb70-11ea-9ed3-592222b70a31.png)

That's approx 30KB/sec log growth rate.

Reply:
Gotcha, I'll bump this back down to 3.4.1

Reply:
With that said, I'm good either way with the version you have set - it's easy to override.  I did raise https://github.com/OpenZWave/Zwave2Mqtt/issues/697 for the issue.

Reply:
Not sure about the lint issue, maybe because this is so large, but we can get it merged as is.